org-quick-peek.el

```diff
@@ -115,11 +115,20 @@
                                                            :num-lines org-quick-peek-show-lines
                                                            :keep-drawers org-quick-peek-show-drawers)))))))
 
-(defun org-quick-peek-agenda-current-item ()
-  "Show quick peek of current agenda item, or hide if one is already shown."
+(defun org-quick-peek-agenda-current-item (&optional logbook)
```

Comment:
Since you added a new command specific to logbook entries, why add an argument here to dispatch to the new command?

Reply:
The new command is a wrapper around `org-quick-peek-agenda-current-item`, passing it entry type `'logbook` -- I'll make the argument `type` rather than `logbook` which probably makes more sense (and allows adding further entry types in future)